docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
+CheckBox is a common component and describes what action will occur when you select it.  
+A CheckBox can only contain an icon, and can be created using style.
+
+![CheckBox1](./media/CheckBox1.png)
+
+## Create with Style
+
+To create a CheckBox applying style, follow these steps:
+
+1. Create CheckBox using the default constructor:
+
+    ```cs
+    CheckBox utilityCheckBox = new CheckBox();
+    ```
+
+2. Apply style to the RadioButton:
+
+    ```cs
+    ButtonStyle utilityStyle = new ButtonStyle
+    {
+        Icon = new ImageViewStyle
+        {
+            Size =  new Size(48, 48),
+            ResourceUrl = new Selector<string>
+            {
+                Normal = "btn_check_off.png",
+                Selected = "btn_check_on_24c447.png",
+                Disabled = "btn_check_off.png",
+                DisabledSelected = "btn_check_on_24c447.png",
+            }
+        }
+    }
+    utilityCheckBox.ApplyStyle(utilityStyle);
+    utilityCheckBox.Size = new Size(48, 48);
+    root.Add(utilityCheckBox);
+    ```
+
+Following output is generated when the CheckBox applying style:
+
+![CheckBox2](./media/CheckBox2.png)
+
+
+## Responding to ClickEvent
+When you click a CheckBox, the CheckBox instance receives a click event.
```

Comment:
Suggest rephrasing to:

When you click a checkbox, the checkbox instance receives a click event.